verigreen-collector-webapp/resources/config.properties

```diff
@@ -25,6 +25,8 @@ jenkins.timeoutInSeconds=3600
 
 # Git configuration
 git.repositoryLocation=<Path to git repository location including .git>
+git.user=<GitUser>
```

Comment:
I need it because the JGIT version I updated to requires it. The system won't crash if it is not available, it will just not pass the credentials along. So it preserves backward compatibility.